packages/clerk-js/src/ui/components/PricingTable/PricingTableDefault.tsx

```diff
@@ -64,3 +83,418 @@ export function PricingTableDefault({
     </InternalThemeProvider>
   );
 }
+
+/* -------------------------------------------------------------------------------------------------
+ * Card
+ * -----------------------------------------------------------------------------------------------*/
+
+interface CardProps {
+  plan: __experimental_CommercePlanResource;
+  planPeriod: __experimental_CommerceSubscriptionPlanPeriod;
+  setPlanPeriod: (p: __experimental_CommerceSubscriptionPlanPeriod) => void;
+  onSelect: (plan: __experimental_CommercePlanResource) => void;
+  isCompact?: boolean;
+  props: __experimental_PricingTableProps;
+}
+
+function Card(props: CardProps) {
+  const { plan, planPeriod, setPlanPeriod, onSelect, props: pricingTableProps, isCompact = false } = props;
+  const isDefaultLayout = pricingTableProps.layout === 'default';
+  const ctaPosition = (isDefaultLayout && pricingTableProps.ctaPosition) || 'top';
+  const collapseFeatures = (isDefaultLayout && pricingTableProps.collapseFeatures) || false;
+  const { id, slug, subscriptionIdForCurrentSubscriber, features } = plan;
+  const totalFeatures = features.length;
+  const hasFeatures = totalFeatures > 0;
+
+  return (
+    <Box
+      key={id}
+      elementDescriptor={descriptors.pricingTableCard}
+      elementId={descriptors.pricingTableCard.setId(slug)}
+      sx={t => ({
+        display: 'flex',
+        flexDirection: 'column',
+        background: common.mergedColorsBackground(
+          colors.setAlpha(t.colors.$colorBackground, 1),
+          t.colors.$neutralAlpha50,
+        ),
+        borderWidth: t.borderWidths.$normal,
+        borderStyle: t.borderStyles.$solid,
+        borderColor: t.colors.$neutralAlpha100,
+        boxShadow: !isCompact ? t.shadows.$cardBoxShadow : undefined,
+        borderRadius: t.radii.$xl,
+        overflow: 'hidden',
+        textAlign: 'left',
+      })}
+      data-variant={isCompact ? 'compact' : 'default'}
+    >
+      <CardHeader
+        plan={plan}
+        isCompact={isCompact}
+        isActivePlan={!!subscriptionIdForCurrentSubscriber}
+        planPeriod={planPeriod}
+        setPlanPeriod={setPlanPeriod}
+      />
+      <ReversibleContainer reverse={ctaPosition === 'top'}>
+        {!collapseFeatures && hasFeatures ? (
```

Comment:
Looks like we were combining the `&&` approach with ternaries. This should just use the ternary for disaplay logic
```suggestion
        {(!collapseFeatures && hasFeatures) ? (
```